scripts/adjust_kernel_config

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+if [ -z "${1}" -o "${1}" == "help" -o "${1}" == "--help" -o "${1}" == "h" -o "${1}" == "-h" ]; then
+  echo -e "A kernel config command must be specified\n"
+  echo -e "$0 menuconfig"
+  exit
+fi
+
+KERNEL_CONFIG_COMMAND="$1"
+
+. config/options linux
+
+${SCRIPTS}/unpack linux
+
+AVAILABLE_KERNEL_CONFIG_COMMANDS="$(make -C ${PKG_BUILD} help | sed -n '/Configuration targets:/,/Other generic targets:/p' | sed -n -E 's/^  ([a-z0-9]+)[ -]*.*/\1/p')"
```

Comment:
Should look like:

`AVAILABLE_KERNEL_CONFIG_COMMANDS="$(ARCH=${TARGET_KERNEL_ARCH} make -C ${PKG_BUILD} ........"`

ARCH is used by the Makefile, and on aarch64 you end up with 
```
Makefile:681: arch/aarch64/Makefile: No such file or directory
make: *** No rule to make target 'arch/aarch64/Makefile'.  Stop.
"menuconfig" is not an available kernel config command
```

I'm guessing it's been a hidden error on the mixed arch devices too.

Reply:
I don't see this behavior
```
$ PROJECT=NXP DEVICE=iMX8 ARCH=arm ./scripts/adjust_kernel_config commands
<snip>
lots of unpacking output
</snip

"commands" is not an available kernel config command

available kernel config commands are:

config
nconfig
menuconfig
xconfig
gconfig
oldconfig
localmodconfig
localyesconfig
defconfig
savedefconfig
allnoconfig
allyesconfig
allmodconfig
alldefconfig
randconfig
yes2modconfig
mod2yesconfig
listnewconfig
helpnewconfig
olddefconfig
tinyconfig
testconfig
```

Reply:
Because "arm" is also a kernel recognized architecture, so the kernel's Makefile finds something to work with at `arch/arm/Makefile`. "aarch64" is not - it wants "arm64"

Reply:
I've updated it to use `kernel_make`